src/pages/concept-list-page/concept-list-resolver.ts

```diff
@@ -6,8 +6,8 @@ import { publisherApiGet } from '../../api/publisher-api';
 const memoizedPublisherApiGetByOrgNr = _.memoize(publisherApiGet);
 
 const mapProps = {
-  concepts: props => conceptCatalogueApiGet(`/begreper?orgNummer=${_.get(props, ['match', 'params', 'catalogId'])}`),
-  publisher: props => memoizedPublisherApiGetByOrgNr(`/publishers/${_.get(props, ['match', 'params', 'catalogId'])}`)
+  concepts: props => conceptCatalogueApiGet(`/begreper?orgNummer=${props.catalogId}`),
+  publisher: props => memoizedPublisherApiGetByOrgNr(`/publishers/${props.catalogId}`)
```

Comment:
Can use object destructuring to avoid useless prosps.

1) concepts: ({ catalogId })=> conceptCatalogueApiGet(`/begreper?orgNummer=${catalogId}`)
2) publisher: ({ catalogId })=> memoizedPublisherApiGetByOrgNr(`/publishers/${catalogId}`)